charts/newrelic-pixie/values.yaml

```diff
@@ -28,3 +28,5 @@ resources:
   requests:
     cpu: 100m
     memory: 250M
+
+tolerations: []
```

Comment:
I am seeing you adding toleration in this values, but it is not currently used by the chart, have you forgot to add it there?
Since it is a new parameter we need to document it as well in the readme

Reply:
Hi, I did not forget, I was trying to follow your standard.

Example of values.yaml files:

1. [newrelic-infrastructure](https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-infrastructure/values.yaml) has values.
2. [new-relic-infra-operator](https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-infra-operator/values.yaml) does not have the values and is the same as I did.

Which one of the two examples I should follow?.

I agree with you that all tolerations should have by default these values as 1.

```
- operator: "Exists"
    effect: "NoSchedule"
  - operator: "Exists"
    effect: "NoExecute"
```
Please let me know and I will correct this.

Thanks,

Reply:
I meant that the chart is not using tolerations and the PR is not adding that

Reply:
Updated the charts with default values using the ones mentioned in my previous comment.

Reply:
Hello, I am sorry I was not clear. The default for the values should be the empty list `[]`. What I meant is that the `toleration` parameter is currently not used by the pixie deployment, therefore setting it in the values is not enough:
https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-pixie/templates/deployment.yaml

Reply:
Hi, thanks for the feedback, I will update the MR to include tolerations for all the required files including pixie

```
deployment.apps/nri-bundle-kube-state-metrics
deployment.apps/kelvin
deployment.apps/vizier-certmgr
deployment.apps/vizier-cloud-connector
deployment.apps/vizier-proxy
deployment.apps/vizier-query-broker
statefulset.apps/vizier-metadata
daemonset.apps/vizier-pem
job.batch/cert-provisioner-job
daemonset.apps/nri-bundle-newrelic-infrastructure
deployment.apps/nri-bundle-newrelic-pixie
deployment.apps/nri-bundle-nri-metadata-injection
deployment.apps/nats-operator
deployment.apps/nri-bundle-nri-prometheus
job.batch/nri-bundle-nri-metadata-injection-admission-create
job.batch/nri-bundle-nri-metadata-injection-admission-patch
deployment.apps/nri-bundle-nri-kube-events
```
And will leave the values as empty list

Reply:
Notice that some of the deployments you listed are not part of this repo (i.e. deployment.apps/kelvin, deployment.apps/vizier-proxy)

Reply:
I generated a new manifest file for Pixie installation with the following options from the automated installer

- Kube state metrics
- Prometheus metrics
- Kubernetes events
- Instant service-level insights, full-body requests, and application profiles through Pixie

Is when I noticed those deployments/stafulsets that also needed tolerations for our use case.

But will modify the PR to match what is there